rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -147,6 +147,43 @@ def test_load_families_correct(self):
         except DatabaseError:
             self.fail("Unable to load families using list ['H_Abstraction', 'pah']")
 
+    def test_kinetics_libraries_have_surface_attributes(self):
```

Comment:
I wonder if we should check every library in the 'Surface' folder, and enery family starting with 'Surface' ?

Reply:
fixed this

Reply:
Thanks for addressing that. I now realize that this sort of testing probably belongs more in the RMG-database tests (the script in `RMG-Py/testing/databaseTest.py` rather than the RMG-Py unit tests.  i.e. in this testing file (`rmgpy/data/kinetics/kinetictsTest.py`) we should be checking that RMG-Py can correctly read, write, access, understand, manipulate a well-formed database.  And in the other place (`RMG-Py/testing/databaseTest.py`) we should be checking that the database is well-formed.

Reply:
I see what you mean, because this goes and checks each entry, but I think a test does belong in rmg-py to make sure that database entries can have these surface attributes.  If someone breaks the way this is currently written, it makes more sense for it to show up in the rmg-py tests.  I can make the more robust (check each surface family entry) in the database tests, but that seems a bit inefficient because you're still loading at least one database twice and running super similar tests.

Reply:
maybe you're right.  I don't think any database tests check thermo libraries yet..